### ISSUE TITLE
NTR: Pupillary hypersensitivity to parasympathomimetic agents (Cranial nerves)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7797]
New term request Pupillary hypersensitivity to parasympathomimetic agents (segal_030124170200): Methacholine; pilocarpine; pupils

**Comment by pnrobinson** [comment]
@MickeySegal
does this term refer to https://pubmed.ncbi.nlm.nih.gov/2330108/? What is the context?

**Comment by MickeySegal** [comment]
We got this from https://www.ncbi.nlm.nih.gov/books/NBK1180/ "Pupillary hypersensitivity to parasympathomimetic agents. Topical administration of methacholine 2.5% or pilocarpine 0.0625% has no observable effect on the normal pupil but causes miosis after approximately 20 minutes in almost all individuals with FD."

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000993
- label: Pupillary hypersensitivity to parasympathomimetic agents
- parents: [Abnormal pupillary function (HP:0007686)](https://hpo.jax.org/app/browse/term/HP:0007686)
- pmid: PMID:20301359|PMID:16754691
- def: A higher than normal tendency of parasympathomimetic agents to induce miosis.
- comment: Topical administration of methacholine 2.5% or pilocarpine 0.0625% has no observable effect on the normal pupil but causes miosis after approximately 20 minutes in almost all individuals with familial dysautonomia.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7797
- synonyms:
